.golangci.yml

```diff
@@ -32,3 +36,6 @@ linters-settings:
     rules:
       - name: unused-parameter
         disabled: true
+  gosec:
+    excludes:
+      - G115 # "Potential integer overflow when converting between integer types"; TODO re-enable eventually
```

Comment:
I personally think that it would be better to either exclude this individually in each line that it's observed today and we know that it's safe, or find a solution for the current instances.
I'm worried that we can keep this forever and hide real issues in the future.

Reply:
That's fair. I can look into using https://github.com/ccoVeille/go-safecast to properly address these issues (i.e. to add the required checks), if adding that dependency seems acceptable? If not, we could implement equivalent functionality in some simpler helpers ourselves.